draft-ietf-scitt-architecture.md

```diff
@@ -324,23 +324,21 @@ In order to accommodate as many Transparency Service implementations as possible
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
 ~~~aasvg
-  .----------.
- |  Artifact  |
-  '----+-----'
-       v
-  .----+----.  .----------.     .-----------.
- | Statement ||  Envelope  +   |  Identity   |
-  '----+----'  '-----+----'    |  Documents  |
-       |             |          '---+--+----'
-        '----. .----'     cose sign |  | cose verify
-              |                     |  |
-              v                     |  |'----------.
-         .----+----.                |  |            |
-        |  Signed   |               |  |            |
-        | Statement +<-------------'|  |            |
-         '----+----'                v  v            |
-              |                 +---+--+--------+   |
-           .-' '--------------->+ Transparency  |   |
+   .----+----.   .----------.   .-----------.
+  | Statement | |  Artifact  | |  Identity   |
+   '----+----'   '-----+----'  |  Documents  |
+        |               |        '---+--+----'
+         '--.   .-----'   cose sign |  | cose verify
+             | |  .----------------'|  |
+             | | |                  |  |
+             v v v                  |  |'----------.
+       .-----+-+-+---.              |  |            |
+      |    Signed     |             |  |            |
+      |   Statement   |             |  |            |
+      |(COSE Envelope)|             |  |            |
+       '-------+-----'              v  v            |
```

Comment:
Per conversation in today's editor's meeting:
I like the change for changing line 342 to include (Cose_Sign1)

I'd like to continue the discussion for an artifact going into Statement. 
- A statement (and its type) is opaque to SCITT and I would assume any statement that is a stand-alone element of content should know what Artifact it's a statement about. 
- Knowing what the artifact is about, as a separate reference to SCITT enables the service communicate to verifiers. The `cwt_clamins.sub` is the property, very similar to the content-type is lifted to communicate and other signing properties. 

Issuers make statements about artifacts, which may be software, hardware, physical goods, they all transcend this triple.

Reply:
I pushed the (COSE_Sign1) change
I found 19 references to "envelope" in the architecture

Reply:
Yeah, this is great actually. When we did this the first time around, as implementors new to the cose landscape, the link with COSE_Sign1 wasn't super clear. We essentially *guessed* that was the right thing. It is really helpful to call out the link between an abstract notion of a statement and its concrete realisation like this.

Reply:
This is a minor nit but I will repeat another time that one could consider: have you ever made a statement about an artifact (software) for which the artifact is *not a necessary input*? Do SCRM tools never talk about the creator of the artifact or their host organization that is logically related, and not an input? We as a group seem focused on SBOM.

This does not change the COSE structure or the document, but these minor changes in the diagram over-specify imply overly specific requirements that lead people astray. I will just repeat this comment, again, as this is why I am going to argue for removal the diagram soon.

Reply:
I would be opposed to removal of this diagram becasue it is an actual reflection of how several existing "Trust Registry" implementations work today, making it easy to adopt the SCITT vision in the Software Supply Chain (SCRM) when the protocol is finalized.

Reply:
Sure, but if we continue to add additional terms and concepts in this discussion, the diagram, and/or spec overall, we run the risk of perpetuating the same problem I have cited before: we are muddling the spec.

Reply:
We do not need to make any changes to the architecture because a "generic" Transparency Service can be instantiated to serve as a "Trust Registry" that contains specific artifacts/payloads and signed statements based on registration policy, very similar to the way a "Registry of Deeds" and a "Registry of Motor Vehicles" are both "Trust Anchors" for different legal artifacts. These could both be implemented as SCITT "Trust Registries" that follow the generic Transparency Service protocol defined in SCITT. This is how I see SCITT being used broadly, in addition to a "Trust Registry" of trusted digital products, such as software and IoT devices.